siphon/simplewebservice/wyoming.py

```diff
@@ -81,6 +81,12 @@ def _get_data(self, time, site_id):
         # Parse metadata
         meta_data = soup.find_all('pre')[1].contents[0]
         lines = meta_data.splitlines()
+
+        # If there are only 26 lines, the station doesn't have a name identified
+        # and we need to insert a record showing this for parsing to proceed.
+        if len(lines) == 26:
```

Comment:
You couldn't instead check:
```python
lines[1].startswith('Station identifier')
```
?

Reply:
Could sure

Reply:
Just seems like it could change from 26 lines for a variety of reasons.

Reply:
Sure, but we'll break on all of  them 😭 
Push coming.